README.md

```diff
@@ -103,3 +103,17 @@ There are a few things to keep in mind when working from an IDE:
 3. Select the file `build.gradle` in the fitnesse folder.
 4. Follow the wizard. Use Java 7 or newer. It should find source and test folders.
 5. Open the Gradle Build tool, select the task `copyRuntimeLibs` and (right-click) mark it as _Execute After Make_.
+
+#### Dockerfile
+
+```
+#To create your image
+$ docker build --tag fitnesse .
```

Comment:
At this form you could simply extend commands below to `docker run -d -p 8001:8001 -v ~/.gradle:/root/.gradle -v $(pwd):/fitnesse -w /fitnesse java:7 ./gradlew run` and achieve the same result. Minus the overhead of building a dedicated docker image.